### ISSUE TITLE
On iOS. the text field cursor behavior has the unexpected move when it's started by the space key long press.

### ISSUE THREAD

**Issue by gki** [flutter/flutter#75555]
## Steps to Reproduce

1. Use TextFormField for the multi line input.
2. Launch app on iOS Simulator, and input texts to the form, at least with 1 Japanese character.
3. Long press the space key and try to move the cursor.

**Notes**
- I cannot reproduce this when I input only English characters **only after the first launch**.
- However, the issue started after I had added just only 1 character of Japanese.
- Also, even if the Japanese characters had been removed to make the text only English, the issue continued.
- Once the issue occurred, hot reload and hot restart doesn't resolve this issue.
- Only `flutter run` can clear the issue.
- I think this is the same issue with https://github.com/flutter/flutter/issues/54413

**Expected results:** 
The cursor should follow the finger move.

**Actual results:** 
1) The cursor sometimes unexpectedly move to the opposite direction of the finger move.
2) Suddenly the cursor stops moving shortly even if finger is moving.

This iOS simulator movie is started recording from just after run `flutter run`. 

As I mentioned above, the first cursor move with only English text has no issue.
But after inputed only 1 Japanese character "あ" at 00:27, the cursor behavior starts to be broken. I moved the mouse slowly on purpose. Please be noticed the mouse arrow direction  and the text cursor direction.
And the broken behavior continued after removing the Japanese character at 01:18.

You can check full source code by the repository below link.

https://user-images.githubusercontent.com/1529390/107115563-d01c9900-68b0-11eb-8d81-3922f2a8e802.mp4

https://github.com/gki/flutter_ios_cursor_test

<details>
  <summary>env</summary>

```
$ flutter analyze 
Analyzing ios_cursor_test...                                            
No issues found! (ran in 2.7s)
```
```

$ flutter doctor -v
[✓] Flutter (Channel stable, 1.22.6, on macOS 11.2 20D64 darwin-x64, locale ja-JP)
    • Flutter version 1.22.6 at /Users/gen/Apps/flutter/flutter
    • Framework revision 9b2d32b605 (2 weeks ago), 2021-01-22 14:36:39 -0800
    • Engine revision 2f0af37152
    • Dart version 2.10.5

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/gen/Library/Android/sdk
    • Platform android-29, build-tools 28.0.3
    • ANDROID_HOME = /Users/gen/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.4)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.0

[!] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.53.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.12.2

[✓] Connected device (1 available)
    • iPhone 11 (mobile) • 36A333B4-F2F0-4CAC-BB1B-11B1FEABAD4E • ios • com.apple.CoreSimulator.SimRuntime.iOS-14-4 (simulator)

! Doctor found issues in 1 category.

```
</details>

**Comment by pedromassangocode** [comment]
<details>
<summary>flutter doctor -v</summary>

```bash
[✓] Flutter (Channel beta, 1.26.0-17.3.pre, on macOS 11.2 20D64 darwin-x64, locale en-AO)
    • Flutter version 1.26.0-17.3.pre at /Users/pedromassango/Code/flutter_beta
    • Framework revision 4b50ca7f7f (4 days ago), 2021-02-04 19:44:27 -0800
    • Engine revision 2c527d6c7e
    • Dart version 2.12.0 (build 2.12.0-259.8.beta)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/pedromassango/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[!] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.1, Build version 12A7403
    ! CocoaPods 1.9.3 out of date (1.10.0 is recommended).
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To upgrade see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] IntelliJ IDEA Community Edition (version 2020.3.2)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 52.2.5
    • Dart plugin version 203.6912

[✓] VS Code (version 1.52.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.18.1

[✓] Connected device (1 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 88.0.4324.150

! Doctor found issues in 1 category.
```

</details>

**Comment by LongCatIsLooong** [comment]
This looks like a duplicate of https://github.com/flutter/flutter/issues/70267. Closing.
